integration/utils/contLoadGenerator.go

```diff
@@ -386,81 +401,152 @@ func (lg *ContLoadGenerator) sendAddKeyTx(workerID int) {
 
 	err = addKeysTx.AddArgument(cadenceKeysArray)
 	if err != nil {
-		lg.log.Error().Err(err).Msgf("error constructing add keys to account transaction")
+		lg.log.Error().Err(err).Msgf("workerID=%d error constructing add keys to account transaction", workerID)
 		return
 	}
 
-	lg.log.Trace().Msgf("creating transaction")
+	lg.log.Trace().Msgf("workerID=%d creating transaction", workerID)
 
 	addKeysTx.SetReferenceBlockID(blockRef).
 		SetProposalKey(*acc.address, 0, acc.seqNumber).
 		SetPayer(*acc.address).
 		AddAuthorizer(*acc.address)
 
-	lg.log.Trace().Msgf("signing transaction")
+	lg.log.Trace().Msgf("workerID=%d signing transaction", workerID)
 	err = acc.signTx(addKeysTx, 0)
 	if err != nil {
-		lg.log.Error().Err(err).Msgf("error signing transaction")
+		lg.log.Error().Err(err).Msgf("workerID=%d error signing transaction", workerID)
 		return
 	}
 
-	lg.sendTx(addKeysTx)
+	lg.sendTx(workerID, addKeysTx)
+}
+
+func (lg *ContLoadGenerator) pushAccountsHappening(workerID int, acc *flowAccount, tx_id string, timeNow time.Time) {
+	lg.log.Trace().Msgf("workerID=%d pushing happening address", workerID)
+	lg.happeningAccounts <- (func() (*flowAccount, string, time.Time) { return acc, tx_id, timeNow })
+}
+
+func (lg *ContLoadGenerator) probeAccountsHappening(workerID int) {
+	channelElements := len(lg.happeningAccounts)
+	lg.log.Trace().Msgf("workerID=%d probing happening address from channel with %d elements", workerID, channelElements)
+	for i := 0; i < channelElements; i++ {
+		acc, tx_id, timeAtSendTx := (<-lg.happeningAccounts)()
```

Comment:
more idomatic to not use underscore for this var name. Will have to change in all locations it's used as well

```suggestion
		acc, txID, timeAtSendTx := (<-lg.happeningAccounts)()
```

Reply:
Sorry, good idea. I look at the logs a lot and they like to use `tx_id`... and that's how it crept in there :-)